SQF/dayz_code/Configs/CfgVehicles/Car/HMMWV.hpp

```diff
@@ -306,9 +306,11 @@ class HMMWV_Base: Car
 };
 
 class HMMWV_DZ: HMMWV_Base {
+	
 	accuracy = 0.32;
 	displayname = $STR_VEH_NAME_HMMWV;
-	maxspeed = 230;
+	displaynameshort = "HMMWV (Wood Camo)";
+	maxspeed = 100;
```

Comment:
You can just remove this and it will inherit its default maxSpeed which is 100. Not sure why vanilla had it as 230, probably from outdated configs.